backend/README.md

```diff
@@ -1,3 +1,12 @@
 # Backend
 
-...
+## Setting up Virtual Environment for Flask
+
+Steps:
+
+1. Set your current directory to the folder named `Image-upload-API` in powershell or vscode terminal
+1. Type `python3 -m venv venv` in the terminal
+1. After installing the necessities type `.\venv\Scripts\activate` activate the venv. On Linux/Mac machines it will be `source venv/bin/activate.sh`
+1. A venv folder should be created, now to install the requirements type `pip3 install -r .\requirements.txt` in the same terminal
+1. To run the Flask app, go to `main.py` in vscode and press the run button at the top right corner. To run it from the terminal, run `python flask-api/main.py`
+1. To exit the virtual environment simply type `deactivate` in the terminal
```

Comment:
I made some nitpicky changes to this file. With markdown you don't have to actually number the list, you can just use "1." for every line and the interpreter will turn it into a numbered list. It makes it easy to go back and add lines later without messing up the number order on the file.

Also for in-line code styling, you can use a single backtick (   ` instead of ``` )